jvm/ml4ir-inference/pom.xml

```diff
@@ -10,6 +10,44 @@
   <artifactId>ml4ir-inference</artifactId>
   <packaging>jar</packaging>
 
+  <profiles>
+    <profile>
+      <id>integration_tests</id>
```

Comment:
so the command to run the integration_test is 
`mvn verify -Pintegration_tests`

I want to exclude unit test from this target, but no success at the moment.

Reply:
Do we run this today?

Reply:
Not in circleCI.

I think it worths a (quick) child story